argus/recipes/base.py

```diff
@@ -21,8 +21,6 @@
 
```

Comment:
add a commit message where you explain what is the purpose of this class and what will solve